ios/FluentUI/Card Nudge/CardNudge.swift

```diff
@@ -177,7 +174,7 @@ public struct CardNudge: View, ConfigurableTokenizedControl {
     }
 }
 
-class MSFCardNudgeStateImpl: NSObject, ControlConfiguration, MSFCardNudgeState {
+class MSFCardNudgeStateImpl: NSObject, ObservableObject, Identifiable, MSFCardNudgeState {
```

Comment:
Are these protocols always going to be implemented by the State classes? Worth creating a typealias to combine both of them?

Reply:
I thought about it, but don't ultimately have a strong preference. If you'd like then I can do that (probably a typealias of `NSObject & ObservableObject & Identifiable` all at once).